pkgs/data/fonts/termsyn/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, mkfontscale }:
+
+stdenv.mkDerivation rec {
+  pname = "termsyn";
+  version = "1.8.7";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/termsyn/termsyn-${version}.tar.gz";
+    sha256 = "15vsmc3nmzl0pkgdpr2993da7p38fiw2rvcg01pwldzmpqrmkpn6";
+  };
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  installPhase = ''
+    install -m 644 -D *.pcf -t "$out/share/fonts"
+    install -m 644 -D *.psfu -t "$out/share/kbd/consolefonts"
+    mkfontdir "$out/share/fonts"
+  '';
+
+  meta = with lib; {
+    description = "Monospaced font based on terminus and tamsyn";
+    homepage = "https://sourceforge.net/projects/termsyn/";
+    license = licenses.gpl2Plus;
```

Comment:
By my reading, this is gpl2Only.

(README.termsyn only mentions "Released under GPLv2 license.")

Reply:
Although gpl2plus also covers GPLv2.0 in licenses.nix I can make the change to be more succinct.

Reply:
gpl2Plus means specifically GPLv2.0 with the or-any-later-version clause. README.termsyn says just "Released under GPLv2 license. (http://www.gnu.org/licenses/gpl-2.0.html)" which doesn't have the or-any-later-version clause.

Reply:
I understand now, sorry for the misunderstanding.